frontend/pages/ManageControlsPage/OSUpdates/components/OSVersionTable/OSVersionTableConfig.tsx

```diff
@@ -61,23 +61,30 @@ export const generateTableHeaders = (teamId: number) => {
       ),
       Cell: ({ row }: IHostCellProps): JSX.Element => {
         const { hosts_count, name_only, version } = row.original;
+        return <TextCell value={hosts_count} />;
+      },
+    },
+    {
+      title: "",
+      Header: "",
```

Comment:
Refactored all "View all hosts" links to follow the same pattern (Having a separate column)